app/views/admin/facilities/index.html.erb

```diff
@@ -42,18 +52,9 @@
               <% end %>
             </h5>
           </div>
-          <div class="col-md-5">
-            <div class="card-details">
-              <i class="fas fa-map-marker-alt light mr-2"></i> <%= facility.street_address %>
-              <%= facility.village_or_colony %>
-              <%= facility.zone %>
-              <%= facility.district %>
```

Comment:
The address in the facilities list seems to be helpful in some cases https://simpledotorg.slack.com/archives/CK95QFJA2/p1580908551060100. @dburka is this good to get rid of?

Reply:
Spoke to Daniel. We are good to axe 'em.